src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './App';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
+      <App />
+    </Router>
```

Comment:
- [ ] You have already imported `BrowserRouter` in App.js. So use this in only one place and remove another to follow the DRY principle and avoid the unnecessary call to the library.